README.md

```diff
@@ -59,6 +59,8 @@ npm install
 
 3. Create a `.env` file in the `client` directory and add any necessary environment variables.
 
+`cp .env.example .env`
```

Comment:
This update is not needed.

Axios.js in the api folder already contains the backend base_url for requests.